tests/driver_mcp2515/main.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2016 OTA keys
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the MCP2515 can driver
+ *
+ * @author      Toon Stegen <toon.stegen@altran.com>
+ *
+ * @}
+ */
+
+#ifndef TEST_MCP2515_SPI
+#error "TEST_MCP2515_SPI not defined"
+#endif
+#ifndef TEST_MCP2515_CS
+#error "TEST_MCP2515_CS not defined"
+#endif
+#ifndef TEST_MCP2515_RESET
+#error "TEST_MCP2515_RESET not defined"
+#endif
+#ifndef TEST_MCP2515_MODE
+#error "TEST_MCP2515_MODE not defined"
+#endif
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "shell.h"
+#include "candev_mcp2515.h"
+
+static candev_mcp2515_t dev;
+struct can_bittiming timing;
+
+static const candev_mcp2515_conf_t conf = {
+    .spi = SPI_1,
+    .cs_pin = CAN2_SS_PIN,
```

Comment:
Causes a compile error as undeclared. Should it be replaced because names have changed or defined by the user?